framework/db/oci/QueryBuilder.php

```diff
@@ -162,6 +162,27 @@ public function addForeignKey($name, $table, $columns, $refTable, $refColumns, $
     }
 
     /**
+     * For each return column a param will be added to $returnParams.
+     * They should have variables bound before executing the query.
+     * @inheritdoc
+     */
+    public function insertReturning($table, $columns, &$params, $returnColumns = null, &$returnParams = null)
+    {
+        $schema = $this->db->getSchema();
```

Comment:
Is it necessary for `$returnParams` to be passed as method argument?
Can't you use column name as param binding name for these?
So its value will be:
```php
[
    ':id' => null
]
```
Then `queryOne()` will return row set just the same as Postgres.

We may check if param ':id' already present amoung query params (which is unlikely for the insert composed by query builder) and throw an exception on it or try to replace it.